test/test_mpi_tree.jl

```diff
@@ -124,6 +124,14 @@ const EXAMPLES_DIR = joinpath(pathof(Trixi) |> dirname |> dirname, "examples", "
     @test_trixi_include(joinpath(EXAMPLES_DIR, "elixir_euler_ec.jl"),
       l2   = [0.061751715597716854, 0.05018223615408711, 0.05018989446443463, 0.225871559730513],
       linf = [0.29347582879608825, 0.31081249232844693, 0.3107380389947736, 1.0540358049885143])
+
+    sol = solve(ode, RDPK3SpFSAL35(), abstol=1.0e-4, reltol=1.0e-4,
```

Comment:
Move this into a proper test (or elixir)? Also, what about testing the 2D and 3D P4est meshes?

Reply:
Hm. I know technically there is no added/different functionality in the other mesh types. However, considering that `P4estMesh` is one of our "production" mesh types, I just would like to avoid surprises when using it with adaptive time stepping, e.g., if someone forgets to use `local_length` somewhere. Or wait - that error would already be triggered by using a non-adaptive-time-stepping setting, right?

Alas, I don't see a clear reason why we'd need to test P4estMesh separately - if neither you do, then feel free to resolve this.

Reply:
> that error would already be triggered by using a non-adaptive-time-stepping setting, right?

Yes, it should. Nevertheless, I added some tests in 76ae70f9 to be sure.

Reply:
Great, thanks!